### PR TITLE
Route corrections and documentation

### PR DESCRIPTION
**Pull request by nate-e-97**
Routes will now return the appropriate error codes and the error codes are now documented in the Swagger UI.
Database functions `get_associate_batch_id` and `get_associate_sf_id` modified to return None if an item isn't found as opposed to failing because of a TypeError.
Weight balancing function modified to handle edge cases of a manager having 0 assigned employees and there only being 1 staging manager.
Dockerfile is added for use with containerization.
Unused files and functions removed, and some code was subject to cleanup.
Added flowchart image for use in the wiki

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 import atexit
 from flask import Flask
 from flask_cors import CORS
+from src.router.models import swot_item, swot_fields, associate_model, associate_model_short, \
```
Comment (rlayneorr) [inline src/app.py]: This is a duplicated import

File: src/external/caliber_processing.py [hunk 2]
```diff
@@ -50,6 +48,9 @@ def assignment_weight(this_batch):
                         _log.info('Batch location: %s', this_batch['location'])
                         _log.info('Manager: %s', manager['_id'])
                         return manager['_id']
+        else:
+            return managers[0]['_id']
```
Comment (rlayneorr) [inline src/external/caliber_processing.py]: why are these else's the same?

**Review by rlayneorr — CHANGES_REQUESTED** [review]

**Comment by nate-e-97** [comment]
Fixed duplicate import in app.py
Changed the return structure in the `assignment_weight` function due to redundant else statements

**Comment by nate-e-97** [comment]
All file changes-
Removed mock api yml file because it was outdated, and because the swagger API generates a json file that fulfills the same purpose
Added flowchart file for use in the wiki
Added Dockerfile for use in containerization
Removed test_associate.py due to its purpose being fulfilled by test_model_associates.py instead

**Review by rlayneorr — APPROVED** [review]

**Pull request merged**